CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Grafana Mimir
 
+* [CHANGE] Mixin: Replace namespace with job label in golang_alerts.
```

Comment:
The mixins have a separate section down in the changelog (look for `### Mixin`). Could you move this there, after we finalize what needs a change for this PR.